### PR TITLE
fix: file id error

### PR DESCRIPTION
**Pull request by spaenleh**
In this PR:
- remove unused id passed to `getUrl` function

### CODE REVIEW

File: src/services/auth/plugins/magicLink/index.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ const plugin: FastifyPluginAsyncTypebox = async (fastify) => {
         query: { lang = DEFAULT_LANG },
       } = request;
       const { url } = body;
-
+      console.log(url);
```
Comment (ReidyT) [inline src/services/auth/plugins/magicLink/index.ts:45]:
```suggestion
```

File: src/services/item/plugins/importExport/service.ts [hunk 2]
```diff
@@ -222,9 +222,9 @@ export class ImportExportService {
         const url = await this.fileItemService.getUrl(actor, repositories, {
           itemId: item.id,
         });
-
+        console.log(url);
```
Comment (ReidyT) [inline src/services/item/plugins/importExport/service.ts:225]:
```suggestion
```

File: src/services/item/plugins/importExport/service.ts [hunk 3]
```diff
@@ -222,9 +222,9 @@ export class ImportExportService {
...
         const res = await fetch(url);
-
+        console.log('Done !');
```
Comment (ReidyT) [inline src/services/item/plugins/importExport/service.ts:227]:
```suggestion
```

**Review by ReidyT — APPROVED** [review]
LGTM 👍

**Comment by spaenleh** [comment]
@ReidyT Thank your for your very careful review !

**Pull request merged**